cypress/e2e/strings/surveyStrings.ts

```diff
@@ -1,19 +1,27 @@
 export const SERISD001_BODY_WEIGHT: string = "SER-ISD-001 - Body Weight";
+export const NEEAR001_BODY_COMPOSITION: string = "NEAAR-001 - Body Composition";
 export const OPEN: string = "OPEN";
 export const CLOSE: string = "Close";
 export const WHAT_IS_YOUR_WEIGHT_TODAY: string = "What is your weight today?";
+export const WHAT_IS_YOUR_BODY_FAT_PERCENTAGE_TODAY: string =
+  "What is your body fat percentage today?";
 export const PLEASE_WRITE_FULL_NUMBER_DECIMAL: string =
   "Please write the full number including the decimal.";
 export const REQUIRED: string = "*Required";
 export const LBS: string = "lbs";
-export const PERCENT50: string = "50%";
-export const PERCENT100: string = "100%";
+export const PERCENT: string = "%";
+export const PERCENT_NUMBER = (number: string) => {
+  return `${number}%`;
+};
```

Comment:
When we have only one statement inside {} we don't need to add them
```suggestion
export const PERCENT_NUMBER = (number: string) => `${number}%`;
```